lib/metrics.js

```diff
@@ -99,6 +100,20 @@ function openwhisk(metrics) {
     } catch (e) {
         // ignore error if not in the context of a docker container
     }
+
+    try {
+        // container os and version read from os-release - https://www.linux.org/docs/man5/os-release.html
+        const osReleaseText = fs.existsSync('/etc/os-release') ?
```

Comment:
Filenames can be turned into constants

Reply:
Good idea - moved these paths to constants.

Taylor